### PR TITLE
fix #61659 fenokulu.net

### LINKED ISSUE #61659: fenokulu.net
### Issue URL (Ads)
 [https://www.fenokulu.net/yeni/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.fenokulu.net%2Fyeni%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.12
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br/>Send Do-Not-Track header, <br/>Remove X-Client-Data header from HTTP requests, <br/>Strip URLs from tracking parameters, <br/>Self-destructing third-party cookies (1440), <br/>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Security:</b><br/> NoCoin Filter List<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish

### PR DESCRIPTION
**Pull request by stanislav-atr**
#61659

### CODE REVIEW

File: TurkishFilter/sections/specific.txt [hunk 1]
```diff
@@ -132,6 +132,9 @@ kalmp3indir.com##a[href^="https://str1.indirdepo.mobi/"]
 mp3indir.rocks###adv
 shunfajinta.com##.widget-content > div[style$=";text-align:center;width:100%;z-index:999999"]
 flowcdn.nl##body > div[style^="z-index:"][style*="position: fixed;"]
+fenokulu.net##a[href^="//www.acilkitap.com/"]
+fenokulu.net##.banner300Soru:upward(div)
```
Comment (Alex-302) [inline TurkishFilter/sections/specific.txt:136]: is not too wide target selector?

Comment (stanislav-atr) [reply TurkishFilter/sections/specific.txt:136]: Static images inside poorly made table layout.

Comment (Alex-302) [reply TurkishFilter/sections/specific.txt:136]: I don't see `.banner300Soru`

Comment (stanislav-atr) [reply TurkishFilter/sections/specific.txt:136]: Another pages, like 
https://www.fenokulu.net/yeni/Soru/kat/14-Fikralar/

Comment (krystian3w) [reply TurkishFilter/sections/specific.txt:136]:
```suggestion
fenokulu.net#?#div.SoruDivSatirDis:has(> div[class^="banner"][class$="Soru"]:not([class*=" "]))
```

upward no need "`#?#`"?

File: TurkishFilter/sections/css_extended.txt [hunk 2]
```diff
@@ -39,6 +39,7 @@ hdfilmcehennemi.com#?##sidebar > .textwidget2:has(> a[href][target="_blank"] > i
 duzcetv.com#?#body > div.container:has(> ins.adsbygoogle)
 marastahaber.com#?#div[style="float: right;"] > a[href][target="_blank"] > video:has(> source[src^="http://marastahaber.com/wp-content/uploads/"][src$="/MaviTekReklam.webm"])
 dizibox.pw#$?##vido-before-video { remove: true; }
+fenokulu.net#?#.banner300Soru:upward(div)
```
Comment (Alex-302) [inline TurkishFilter/sections/css_extended.txt:42]: add also the rule for platforms without ExtCSS

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: TurkishFilter/sections/css_extended.txt (hunk 2 above)
Comment (stanislav-atr) [reply TurkishFilter/sections/css_extended.txt:42]: Mobile layout has no such elements, is it necessary?

Comment (krystian3w) [reply TurkishFilter/sections/css_extended.txt:42]: `upward` works on latest Windows `.exe` / macOS / `.apk` (full) app?

**Review by krystian3w — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**